tools/cldr-code/src/main/java/org/unicode/cldr/util/VoteResolver.java

```diff
@@ -1136,12 +1136,12 @@ private void resolveVotes() {
          */
         if (sortedValues.size() == 0) {
             if (baselineValue != null) {
-                winningValue = baselineValue;
+                setWinningValue(baselineValue);
```

Comment:
this is the change that fixes the bug I was encountering

notice that it's a fallback for when there are no (unconflicted) votes